test/app-forms.spec.js

```diff
@@ -183,7 +183,7 @@ describe('forms that have caused bugs', () => {
     expect(twoTels.report).to.nested.deep.include({
       form: 'tel',
       'fields.contacts.n_contacts': '',
-      'fields.contacts.contact_repeat': [{ text: '17786041234' }, { text: '17786042345' }],
+      'fields.contacts.contact_repeat': [{ text: '+17786041234' }, { text: '+17786042345' }],
```

Comment:
Out of everything, this is the one thing I do not understand.  :facepalm:   For the life of me, I cannot figure out why it behaves like this or what made it change.  I have poured many hours of debugging into trying to figure this out, without success...

Hoping someone else here has an idea about what might be going on!

Reply:
weird. seems like reasonable behavior though

Reply:
Yeah, honestly have the `+` is actually the behavior I would expect since it is the result of the phone-widget calling the [`phoneNumber.normalize` function](https://github.com/medic/cht-core/blob/814ed5cd488d783784959e60c4066a665319accb/webapp/src/js/enketo/widgets/phone-widget.js#L17).  It seems like this should happen all the time, though, whenever a phone number is set. So I don't understand why when only one phone number is provided earlier in this test that the `+` is NOT added....  I cannot recreate similar behavior either in the actual CHT, or just when running cht-form independent of the harness (it always adds the `+`).